src/features/advanced/prodq-hide-government-links.ts

```diff
@@ -0,0 +1,8 @@
+import { applyScopedClassCssRule } from '@src/infrastructure/prun-ui/refined-prun-css';
+import css from '@src/utils/css-utils.module.css';
+
+function init() {
+  applyScopedClassCssRule('PRODQ', C.Link.link, css.hidden);
```

Comment:
Right, so this will definitely work, but let's err on the safe side and scope it more specifically. Something like:
```css
tr td:nth-child(3) .${C.Link.link}
```